src/linalg/exp.rs

```diff
@@ -0,0 +1,479 @@
+//! This module provides the matrix exponent (exp) function to square matrices.
+//!
+use crate::{
+    base::{
+        allocator::Allocator,
+        dimension::{DimMin, DimMinimum, DimName},
+        DefaultAllocator,
+    },
+    try_convert, ComplexField, MatrixN, RealField,
+};
+
+// https://github.com/scipy/scipy/blob/c1372d8aa90a73d8a52f135529293ff4edb98fc8/scipy/sparse/linalg/matfuncs.py
+struct ExpmPadeHelper<N, R>
```

Comment:
Whenever we have a square matrix, we use `D` as the dimension type parameter in nalgebra. So every occurrence of `R` should be replaced by `D` in this file.

Reply:
Done!